draft-ietf-git-github-wg-configuration.md

```diff
@@ -132,19 +133,16 @@ There should likely be an API to specify that there were personnel changes.
 ## Working Group Closing
 
 When a working group is closed, the team with administrative access would be removed and
-the owner list would be returned to its initial composition. The organization summary and
-the repositories within the organization would be updated to indicate that they are no
-longer under development.
+the owner list would be returned to the Secretariat and current ADs at the time of closing.
+The organization summary and the repositories within the organization would be updated to
+indicate that they are no longer under development.
 
 ## Creation of Document Repository {#repo_create}
 
 There are many different scenarios and configurations where it might be useful to have
 automation or established administrative conventions for repositories within WG
 organizations, such as:
 
-   - Creating a new repository for an individual draft that is at the discretion of the
-   WG chair;
```

Comment:
I would prefer if we keep this and say "Creating a new repository for an individual draft (at the discretion of the WG chair);"

Reply:
That works for me. I don't feel strongly about this. Would you mind replying to Barry with respect to this comment?